src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const NavBar = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+          to="/"
+        >
+          Home
+        </NavLink>
```

Comment:
move repeated code to separate component

Reply:
I added there and in previous 2 tasks, but it doesn't pass tests because of it